docs/release-notes/3539-replace-core-experiment-with-new-api.txt

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Master: removes endpoints (/:experiment_id, /:experiment_id/checkpoints, /:experiment_id/config,
+   /:experiment_id/model_def, /:experiment_id/preview_gc, /:experiment_id/summary, and
+   /:experiment_id/metrics/summary) from the legacy API. This data is available in the GRPC API
```

Comment:
nit we could say GRPC and grpc-gateway (or just grpc? 🤔 ) backed rest api. although I'm not sure if we're exposing ports for direct grpc communication or if we wan't to advertise it yet.

Reply:
:+1: will change to REST API